packages/backend/src/routers/organizationRouter.ts

```diff
@@ -55,10 +55,12 @@ organizationRouter.get('/projects', isAuthenticated, async (req, res, next) =>
 organizationRouter.post('/projects', isAuthenticated, async (req, res, next) =>
     projectService
         .create(req.user!, req.body)
-        .then((results) => {
+        .then((jobUuid) => {
```

Comment:
so this is currently breaking the UI and we can't update the UI until we merge @nathaliabj 's work?

Reply:
correct